invenio_cli/helpers/commands.py

```diff
@@ -334,11 +343,6 @@ def _lock_python_dependencies(self, pre):
 
     def containerize(self, pre, force, install, stop):
```

Comment:
Remove also the parameter since it's not used. No?
```suggestion
    def containerize(self, pre, force, install):
```